docs/quickstart.md

```diff
@@ -0,0 +1,391 @@
+
+```
+$ pip install Cython
+```
+
+Or via `conda`:
+
+```
+$ conda install Cython
+```
+
+Then run:
+
+```
+$ python setup.py install
+```
+
+or if you are developing `primer3-py` enhancements:
+
+```
+$ python setup.py build_ext --inplace
+```
+
+We recommend running `setup.py` with either `build_ext --inplace` or
+`develop` rather than `install` if you are testing development builds.
+`build_ext --inplace` will build the Cython and C API extensions in the
+package directory without copying any files to your local environment
+site-packages directory (so you can import and run tests from within the
+package) and `develop` will build in place and then put symlinks in your
+site packages directory (this will allow Primer3-py)
+
+NOTE: If you're attempting to build on Windows, please review the `primer3`
+documentation regarding environment requirements. You'll need to install
+the latest version of the TDM-GCC MinGW Compiler if building in a
+`MinGW / Mingw-w64` environment: [TDM-GCC MinGW Compiler](https://jmeubank.github.io/tdm-gcc/)
+
+## Testing
+
+Every commit pushed to
+[the primer3-py GitHub repo](https://github.com/libnano/primer3-py) is tested to
+ensure it builds properly and passes our unit testing framework as a GitHub action
+
+If you'd like to run the tests yourself, we suggest the following workflow:
+
+```
+$ git clone https://github.com/libnano/primer3-py
+$ cd primer3-py
+$ python setup.py build_ext --inplace
+$ pytest
+```
+
+NOTE: `pip` / `conda` install `pytest` if not in your environment
+
+
+## Contributing
+
+Contributions are welcomed via pull requests.
+
+Contact the `primer3-py` maintainers prior to beginning your work to make sure
+it makes sense for the project.
+
+### Forking
+
+A forking workflow is preferred for all pull requests.
+
+### Branch naming
+
+Branch naming is preferred to use the format:
+
+```
+<GitHub user-name>-<short keyword description of change>
+```
+
+Keep branch names not too long.  A good example would be for the user `grinner`
+for a documentation update for the 1.0.0 staging branch:
+
+```bash
+$ git checkout -b grinner-docs-update-1.0.0-pass-01
+```
+
+With the trailing 01 indicative of it being part of several potential
+
+Another example pass that focuses on code clarity comments would be:
+
+```bash
+$ git checkout -b grinner-code-clarity-and-comments
+```
+
+### Development
+
+Development requires the use of C Python 3.8+, `pytest <https://docs.pytest.org>` and
+`pre-commit <https://pre-commit.com>` as they are used to build and run primer3-py
+code CI in the GitHub Action.
+
+Install these dependencies in your python development environment
+(`virtualenv`, `conda`, etc):
+
+```bash
+$ pip install cython pre-commit pytest
+# or
+$ conda install cython pre-commit pytest
+```
+
+Install `pre-commit` in repo the with:
+
+```bash
+$ pre-commit install
+```
+
+To ensure the git hook is excecuted on every commit.
+
+### Pull Requests
+
+Pull Requests should meet the following requirements:
```

Comment:
I would be in favor of adding a line item for updating and/or adding test coverage for any changes / additions

Reply:
+1 on test coverage line item